pkg/plugins/policies/meshtimeout/plugin/testdata/basic_inbound_listener.golden.yaml

```diff
@@ -27,7 +27,7 @@ filterChains:
             route:
               cluster: backend
               timeout: 5s
-      statPrefix: "inbound_127_0_0_1_80"
```

Comment:
Looks like changes in `plugins/policies/` are not related to the change

Reply:
removed